modules/ibi-image-based-install-cluster-guide.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-install/ibi-understanding-image-based-install.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-install-hub-cluster-guide_{context}"]
+= Cluster guidelines for an image-based installation and deployment
+
+For a successful image-based installation and deployment, see the following guidelines.
+
+== Cluster guidelines
+
+* If you are using {rh-rhacm-first}, to avoid including any {rh-rhacm} resources in your seed image, you need to disable all optional {rh-rhacm} add-ons before generating the seed image.
+
+== Seed cluster guidelines
+
+* If your cluster deployment at the edge of the network requires a proxy configuration, you must create a seed image from a seed cluster featuring a proxy configuration. The proxy configurations do not have to match.
+
+* If you set an maximum transmission unit (MTU) in the seed cluster, you must set the same MTU value in the static network configuration for the image-based configuration ISO.
+
+* Your {sno} seed cluster must have a shared `var/lib/containers` directory for precaching images during an image-based installation. For more information see the section _Configuring a shared container directory between ostree stateroots_.
+
+* Create a seed image from a {sno} cluster with the same hardware as your bare-metal host, and reflects your target cluster configuration. The seed cluster must reflect your target cluster configuration for the following items:
+
+** CPU topology
+*** Number of CPU cores
+*** Tuned performance configuration, such as number of reserved CPUs
+
+** IP version
++
+[NOTE]
+====
+Dual-stack networking is not supported in this release.
+====
+
+** Disconnected registry
```

Comment:
A different disconnected registry can be used. Perhaps we should re-phrase (or is it clear) ?

Reply:
@eranco74 - any thoughts on this one?

Reply:
Ah, @achuzhoy is right, this should mention that the seed cluster doesn't need to use the same mirror registry configuration as the target cluster.

Reply:
Would adding a note like this help? I'm unsure of the technical details here:
````
"NOTE: If the target cluster uses a disconnected registry, your seed cluster must use a disconnected registry. The registries do not have to be the same. "
````

Reply:
Looks good to me.